Mlem/App/Utility/Extensions/Content Models/ReportableProviding+Extensions.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ReportableProviding+Extensions.swift
+//  Mlem
+//
+//  Created by Sjmarf on 12/08/2024.
+//
+
+import MlemMiddleware
+
+extension ReportableProviding {
+    func showReportSheet(communityContext: (any CommunityStubProviding)? = nil) {
+        NavigationModel.main.openSheet(.report(self, community: communityContext))
+    }
+    
+    func reportAction(communityContext: (any CommunityStubProviding)? = nil) -> BasicAction {
+        .init(
+            id: "report\(uid)",
+            isOn: false,
+            label: "Report",
+            color: Palette.main.negative,
+            isDestructive: true,
+            icon: Icons.moderationReport,
+            callback: true ? { self.showReportSheet(communityContext: communityContext) } : nil
```

Comment:
*nit*

Ternary unnecessary since condition always `true`

```suggestion
            callback: { self.showReportSheet(communityContext: communityContext) }
```